app/ModelSerializers/SerializerRegistry.php

```diff
@@ -229,8 +229,9 @@ private function __construct()
         $this->registry['Presentation'] =
         [
             self::SerializerType_Public => PresentationSerializer::class,
-            self::SerializerType_Private => AdminPresentationSerializer::class,
+            //self::SerializerType_Private => AdminPresentationSerializer::class,
```

Comment:
Temporarily commented out until we define a particular selection criteria or decide to unregister this superclass

Reply:
please create. a new SerializerType_
like
SerializerType_Admin

Reply:
new type created as requested: SerializerType_Admin_Voteable